packages/nocodb/src/lib/controllers/audit.ctl.ts

```diff
@@ -30,6 +30,15 @@ export async function commentList(req: Request<any, any, any>, res) {
   );
 }
 
+export async function commentUpdate(req: Request<any, any>, res) {
+  res.json(
+    await auditService.commentUpdate({
+      auditId: req.params.auditId,
+      body: req.body,
+    })
```

Comment:
Can anyone update the comment?

Reply:
Only the owner of the comment. Just pushed the code.